validator/accounts/v2/testing/BUILD.bazel

```
@@ -0,0 +1,12 @@
+load("@prysm//tools/go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["mock.go"],
+    importpath = "github.com/prysmaticlabs/prysm/validator/accounts/v2/testing",
+    visibility = [
+        "//validator:__pkg__",
+        "//validator:__subpackages__",
+    ],
+    deps = ["@com_github_dustinkirkland_golang_petname//:go_default_library"],
```

```suggestion
    deps = ["@com_github_dustinkirkland_golang_petname//:go_default_library"],
    testonly = 1,
```
Please mark as testonly